add python REST API example (#1019)
dataform-co/dataform: content/docs/dataform-web/api/index.md

  "runLogUrl": "https://app.dataform.co/#/1234/run/5678"
}
```

## Triggering a Dataform schedule from a 3rd party orchestration tool

Using the REST API it's possible to trigger Dataform schedules from a third party orchestration tool, like Airflow or Luigi.

> The following example, written in Python, shows how you could initiate a schedule, check it's status every 10 seconds, and exit when the schedule either succeeds or fails.

Reviewer: ```suggestion
The following example, written in Python, shows how you can trigger a schedule, check its status every 10 seconds, and exit when the schedule finishes.
```

```python
import requests

> import time

Reviewer: import json

base_url='https://api.dataform.co/v1/project/<PROJECT_ID>/run'
headers={'Authorization': 'Bearer <API_TOKEN>'}

> run_create_request='{ "scheduleName": "<SCHEDULE_NAME>" }'

Reviewer: run_create_request={"environmentName": "<ENVIRONMENT_NAME>", "scheduleName": "<SCHEDULE_NAME"}

> response = requests.post(base_url, data=run_create_request, headers=headers)

Reviewer: ```suggestion
response = requests.post(base_url, data=json.dumps(run_create_request), headers=headers)
```

run_url = base_url + '/' + response.json()['id']

response = requests.get(run_url, headers=headers)

> print(response.json())

Reviewer: rm this line?
